samples/Basic_NFC/app/application.cpp

```diff
@@ -0,0 +1,90 @@
+#include <user_config.h>
+#include <SmingCore/SmingCore.h>
+#include "MFRC522.h"
```

Comment:
@icodk May be you should change also the include to point to the right header file
```
#include <Libraries/MFRC522/MFRC522.h>
```